hbase-common/src/main/java/org/apache/hadoop/hbase/trace/HBaseSemanticAttributes.java

```diff
@@ -38,14 +38,13 @@
   public static final AttributeKey<String> TABLE_KEY = AttributeKey.stringKey("db.hbase.table");
   public static final AttributeKey<List<String>> REGION_NAMES_KEY =
     AttributeKey.stringArrayKey("db.hbase.regions");
-  public static final AttributeKey<String> RPC_SERVICE_KEY =
-    AttributeKey.stringKey("db.hbase.rpc.service");
-  public static final AttributeKey<String> RPC_METHOD_KEY =
-    AttributeKey.stringKey("db.hbase.rpc.method");
+  public static final AttributeKey<String> RPC_SYSTEM = SemanticAttributes.RPC_SYSTEM;
```

Comment:
Do we still need to maintain this alias in HBaseSemanticAttributes? I'm neutral， just asking...

Reply:
I prefer it here because it keeps the scope of our tracing interface collected in one place. However, I'm also neutral here and don't mind importing `SemanticAttributes` directly instead of using this alias.